### ISSUE TITLE
JSON syntax highlighting on Mac

### ISSUE THREAD

**Issue by macino** [a85/POSTMan-Chrome-Extension#9623]
**Describe the Issue**

The syntax highlight for json in body or javascript in pre req script and formatting and syntax highlight does not work.

**Steps to Reproduce**

Request:
- Open request
- write some body or pre req script
- RESULT: syntax highlight is not used

Response:
- Send a request
- json response is not highlighted or formatted

**Screenshots or Videos**

![Snímka obrazovky 2021-02-17 o 16 57 18](https://user-images.githubusercontent.com/522323/108230597-39b66600-7141-11eb-82f1-eabe3810187c.png)

**Environment Information**

 - Platform Type: Native Desktop App
 - Postman Version: 8.0.6
 - Operating System: macOS Big Sur 11.2.1

**Additional Context**

It worked in previous versions of postman (eg. 7x) on the same macOS version.

**Comment by kamalaknn** [comment]
@macino Could you confirm that this issue exists even after you restart your app? If you continue seeing this issue can you share a screenshot of any errors you see in `View -> Developer -> Show DevTolls (Current View)`?

**Comment by macino** [comment]
@kamalaknn So I went a bit longer road. As I restarted the app it was back in 7.36.5 version (don't know how). The update in application failed so I downloaded the new version and replaced the old one in apps.

As the new one started, syntax highlight is ok on both sites. Tried restarting the app and it's still on version 8.0.6 with everything working as I am concerned. So maybe the issue was in the updating process.

Thank you.

edit:

Maybe I should mention the first upgrade process. The in-app upgrade didn't work either, so I downloaded it but instead of replacing the app I deleted the old version and copied the new one. So the deletion vs replacement is the only difference afaik.
